test/ts/logs/log_utils.ts

```diff
@@ -1,6 +1,8 @@
 import * as ABIDecoder from "abi-decoder";
 import * as BigNumber from "bignumber.js";
 import * as _ from "lodash";
+import * as Web3 from "web3";
+import { SolidityType } from "../../../types/common";
```

Comment:
Do you want to separate these out, as groups? I usually like to keep them separate, because it gets so chaotic when there are lots of imports.

```typescript
// External
import * as ExternalLib from "external";

// Common
import { CommonThing } from "../../relative/common";
```